ext/auto_flush.c

```diff
@@ -28,7 +29,7 @@ ZEND_RESULT_CODE ddtrace_flush_tracer(bool force_on_startup, bool collect_cycles
 
     if (zend_hash_num_elements(Z_ARR(trace)) == 0) {
         zend_array_destroy(Z_ARR(trace));
-        ddtrace_log_debug("No finished traces to be sent to the agent");
+        LOG(Info, "No finished traces to be sent to the agent");
```

Comment:
I have a feeling that you set `Info` where I'd set `Debug`.  
So as setting `DD_TRACE_DEBUG` to true is allowing info and warning logs, it's basically the same result. But for consistency with other tracers (at least the one I know), I would have set `Debug`. Because again, it could help in the future if we have a centralized UI (or shared saved views) to use the same levels across tracers. That said I haven't checked all tracers so can't be 100% sure. 
And this can be reworked later as when `DD_TRACE_DEBUG` is set to false we don't get those logs